.github/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ## 0.13.0
 
+1. [EFB/PERF] Harmonize takeoff and landing performance calculators - @FoxtrotSierra6829 (FoxtrotSierra)
```

Comment:
New items should go at the end of the list for 0.13.0.

Reply:
Oops okay will fix